umap/templates/500.html

```diff
@@ -1,8 +1,11 @@
 <!DOCTYPE html>
-<html>
-<head>
+<html lang="en">
+  <head>
     <title>Ooops</title>
     <meta charset="utf-8">
+    <meta name="description"
+          content="uMap lets you create maps with OpenStreetMap layers in a minute and embed them in your site.">
```

Comment:
Shouldn't this be translated ? I think we already have a string like this in transifex.